src/display/annotation_layer.js

```diff
@@ -239,6 +239,38 @@ class AnnotationElement {
     return container;
   }
 
+  /**
+   * Create areas for the quadPoints
+   *
+   * @private
+   * @memberof AnnotationElement
+   */
+  _createAreas() {
```

Comment:
Should this not take a `ignoreBorder` parameter, similar to the `createContainer` method?